libs/datamodel/core/tests/attributes/relations_new.rs

```diff
@@ -20,7 +20,7 @@ fn relation_happy_path() {
     }
     "#;
 
-    let schema = parse(dml);
+    let schema = parse(dml).datamodel;
```

Comment:
You could avoid all those diffs if you move the `.datamodel` into the `parse` function instead. If you want to to check the warnings sometimes you should probably introduce an additional function called `parse_diagnostics`. That would make this PR much more digestable.